### PR TITLE
Octavia backport

### PR DESCRIPTION
**Pull request by codyhammock**
Brings in octavia-related code from the latest version of upstream so that we can use it in our Train branch

### CODE REVIEW

File: tools/kolla-ansible [hunk 1]
```diff
@@ -123,28 +123,29 @@ Options:
...
-    deploy-containers   Only deploy and start containers (no config updates or bootstrapping)
-    post-deploy         Do post deploy on deploy node
-    pull                Pull all images for containers (only pulls, no running container changes)
-    reconfigure         Reconfigure OpenStack service
-    stop                Stop Kolla containers
-    certificates        Generate self-signed certificate for TLS *For Development Only*
-    upgrade             Upgrades existing OpenStack Environment
-    upgrade-bifrost     Upgrades an existing bifrost container
-    genconfig           Generate configuration files for enabled OpenStack services
+    prechecks            Do pre-deployment checks for hosts
```
Comment (msherman64) [inline tools/kolla-ansible:126]: What caused this big diff?

Comment (codyhammock) [reply tools/kolla-ansible:126]: whitespace, maybe?

Comment (diurnalist) [reply tools/kolla-ansible:126]: `ocavia-certificates` was added as a command, and it became the longest command in the first column, so all the descriptions had to shift, I think.

**Review by diurnalist — COMMENTED** [review]
Retroactive LGTM 👍🏻

**Pull request merged**